src/glue/v2/communication.cpp

```diff
@@ -152,9 +153,10 @@ storage::v3::Result<communication::bolt::Vertex> ToBoltVertex(const storage::v3:
 
 storage::v3::Result<communication::bolt::Edge> ToBoltEdge(const storage::v3::EdgeAccessor &edge,
                                                           const storage::v3::Storage &db, storage::v3::View view) {
-  auto id = communication::bolt::Id::FromUint(edge.Gid().AsUint());
-  auto from = communication::bolt::Id::FromUint(edge.FromVertex().Gid().AsUint());
-  auto to = communication::bolt::Id::FromUint(edge.ToVertex().Gid().AsUint());
+  // TODO(jbajic) Fix bolt communication
+  auto id = communication::bolt::Id::FromUint(0);
+  auto from = communication::bolt::Id::FromUint(1);
+  auto to = communication::bolt::Id::FromUint(2);
```

Comment:
I would say let's use only `0`, no real meaning for `1` and `2`, then why do use them?